web.py

```diff
@@ -141,5 +148,5 @@ def check_room_code(code):
     return code in games.keys()
 
 
-if __name__ == '__main__':
-    socketio.run(app, debug=True, host=u'0.0.0.0')
+if __name__ == "__main__":
+    socketio.run(app, debug=False, host="0.0.0.0")
```

Comment:
```suggestion
    socketio.run(app, debug=True, host="0.0.0.0")
```

This should all probably be reading in parameters from a config file or something, but let's keep the original default value.